File: solutions/python/difference-of-squares/1/difference_of_squares.py
def square_of_sum(number):
    resultadoSquare = 0
    for i in range(1, number+1):
        resultadoSquare += i
    return resultadoSquare ** 2
        


def sum_of_squares(number):
    resultadoSuma = 0
    for i in range(1, number+1):
        resultadoSuma += i ** 2
    return resultadoSuma

def difference_of_squares(number):
    diferencia = square_of_sum(number) - sum_of_squares(number)
    return diferencia

File: solutions/c/difference-of-squares/1/difference_of_squares.c
#include "difference_of_squares.h"


unsigned int square_of_sum(unsigned int n) {
    unsigned int valor1 = 0;
    for (unsigned int calculo = 1; calculo <= n; calculo++) {
        valor1 += calculo;
    }
    return valor1 * valor1;
}
unsigned int sum_of_squares(unsigned int n){
    unsigned int valor2 = 0;
    for (unsigned int calculo = 1; calculo <= n; calculo++) {
        valor2 += calculo * calculo;
        
    }
    return valor2 ;
}

unsigned int difference_of_squares(unsigned int n) {
    unsigned int calculo1 = square_of_sum(n);
    unsigned int calculo2 = sum_of_squares(n);
    return calculo1 - calculo2;
    }



